tests/link/jax/test_random.py

```diff
@@ -165,6 +165,22 @@ def test_random_updates(rng_ctor):
             "logistic",
             lambda *args: args,
         ),
+        (
+            aer.lognormal,
+            [
+                set_test_value(
+                    at.lvector(),
+                    np.array([0, 0], dtype=np.int64),
```

Comment:
Sees `scipy`'s implementation does not accept a `loc` param ? https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.lognorm.html

Reply:
It seems to work with something like `scipy.stats.lognorm(1, loc=10).rvs()`.